docs/migrating-from-mdl.md

```diff
@@ -0,0 +1,318 @@
+# Migrating from Material Design Lite
+
+Material Components for the web (MDC-Web) is the successor to the Material Design Lite (MDL) project.
+
+While the philosophy behind the two projects is quite similar, migrating to MDC-Web requires a number of changes, from
+class names to different DOM structures. In addition, there are several choices to be made regarding component
+initialization, how to depend on MDC-Web, and theming/styling mechanisms.
+
+This document attempts to summarize and guide you through the work involved. Let’s get started!
+
+> **Note:** If you’re thinking of migrating your application to MDC-Web, please bear in mind that it’s still in an alpha
+state and thus not recommended for production.
+
+
+## Depending on MDC-Web
+
+MDL is distributed on NPM, Bower, and through its own CDN. MDC-Web is currently only available on NPM.
+
+While MDL is an all-in-one solution, MDC-Web has been designed to be modular, allowing you to make the choice of either
```

Comment:
Is "all-in-one" solution the right term here? Do we want to say something like "While MDL is a singular, universal library consisting of all components and styles", etc.